source/_code.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <button aria-controls="<%= name %>-snippets"
+          aria-expanded="false"
```

Comment:
I spiked it out and got it working. 

I appreciate the idealism, but think treating the presence of *any* CSS as a progressive enhancement is unusual and probably not worth the tradeoffs for maintainability and readability.